### ISSUE TITLE
Internet Connection test failing

### ISSUE THREAD

**Issue by m1lkman** [Romanitho/Winget-AutoUpdate#204]
I have fresh machines with IE not enabled or configured, need `-UseBasicParsing` added to line 13 to allow this function to work. Invoke-WebRequest needs this. Tried opening a pull request but edit is disabled.

**Comment by Romanitho** [comment]
Hello. I have fixed that, but need to release. Thanks for feedback

**Comment by Romanitho** [comment]
by the way, you should be able to PR

**Comment by Romanitho** [comment]
Yes, really sorry about that. On my testings I didn't get the case.  
Needs to be manually reinstalled :(
